Several site updates from April and May

NeilHanlon opened the pull request:
See commit log for details

ressonix commented:
What does this achieve, exactly? I see you opened an issue about 404 in non-English languages when going to "Posts", but this PR doesn't resolve that.

domingoruiz commented:
@ressonix This pull request does not resolve the issue, but this pull request needs to be done. Absolute paths "/", "/ posts /" are being used. These urls are not correct on a multilanguage site, hence this pull. In fact, before this pull, the Posts button did not work, at least a 404 already appears. This is a necessary first step to resolve the issue.

ressonix commented:
In that case, since everything appears to be working okay with it, I'll go ahead and merge this. Thanks!

dependabot[bot] committed:
Bump next in the npm_and_yarn group across 1 directory

Bumps the npm_and_yarn group with 1 update in the / directory: [next](https://github.com/vercel/next.js).


Updates `next` from 15.2.3 to 15.2.4
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v15.2.3...v15.2.4)

---
updated-dependencies:
- dependency-name: next
  dependency-version: 15.2.4
  dependency-type: direct:production
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>

Commit:
Fixing 404s on five pages

Signed-off-by: Chris "Not So" Short <cshort@ciq.com>

brianclemens committed:
s/fathom/plausible

NebraskaCoder committed:
Enhance GitHub workflows to trigger on merge_group events for Playwright and Validate tests

NebraskaCoder committed:
Update packages and removed Babel dependency, Updated ESLint configuration to support TypeScript and Next.js, and enhance Jest configuration, added new tests

NebraskaCoder committed:
Update Jest configuration to target ES2022, enable loose mode, and refine transformIgnorePatterns for CSS modules.

NebraskaCoder committed:
Update package-lock.json to reflect dependency upgrades, including Babel and related packages, and add missing license information for several modules.

NebraskaCoder committed:
Refine Jest configuration by removing redundant transformIgnorePatterns and adding a comment to clarify handling of CSS modules.

NebraskaCoder committed:
Refactor download page components and add DownloadContent

- Simplified Jest configuration by merging object properties.
- Refactored the Download page to utilize a new DownloadContent component for better organization.
- Updated NavItem component to streamline link rendering.
- Introduced new CloudImage and DefaultImage components with improved structure and localization support.
- Enhanced table components with localized headers and cell content.
- Added internationalization support for various UI elements in the download section.

NebraskaCoder committed:
Refactor sorting logic in CloudImage table headers for consistency

- Updated sorting condition in VersionHeader, RegionHeader, and ArchitectureHeader components to use strict equality (===) instead of loose equality (==) for improved code reliability.

FoggyMtnDrifter committed:
New translations en.json (Chinese Simplified)

FoggyMtnDrifter committed:
New translations en.json (Chinese Traditional)

NebraskaCoder committed:
Merge pull request #138 from rocky-linux/feature/upgrades

NebraskaCoder committed:
Merge pull request #139 from rocky-linux/40-feature-add-i18n-support-for-cloud-image-datatable-and-dialog-component

Commit:
new article for RL Support for RISC-V in RL10

Commit:
fix suggested by Louis

NeilHanlon committed:
cleanup and add details on alternative architecture status
